### PR TITLE
Added a sub-domain Record.

### PR DESCRIPTION
**Pull request by sahilpatel09**

### CODE REVIEW

File: domains/sahil.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal Porfolio Records Add.",
+  "repo": "https://github.com/sahilpatel09/sahilpatel09.github.io",
+  "owner": {
+    "username": "sahil",
```
Comment (MaskDuck) [inline domains/sahil.json:5]: gh username pls?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by sahilpatel09 — COMMENTED** [review]
Added gh username

File: domains/sahil.json (hunk 1 above)
Comment (sahilpatel09) [reply domains/sahil.json]: I have added the github username. Sorry forgot it.

**Review by MaskDuck — APPROVED** [review]
acceptable

**Pull request merged**